wavefront_cli/cli.py

```diff
@@ -36,12 +36,12 @@
 
 def main():
     """Main CLI entrypoint."""
-    import commands
+    from . import commands
```

Comment:
Is there any particular reason for placing this import statement inside the function instead of top level?